charts/frigate/Chart.yaml

```diff
@@ -11,6 +11,10 @@ home: https://github.com/blakeblackshear/blakeshome-charts/tree/master/charts/fr
 icon: https://github.com/blakeblackshear/frigate/blob/master/docs/static/img/frigate.png
 sources:
   - https://github.com/blakeblackshear/frigate
+dependencies:
+- name: common
+  version: 3.0.0
+  repository: https://k8s-at-home.com/charts/
```

Comment:
I am not sure if we want to have the dependencies on this library chart. Might be better to copy over any logic/code needed to make the ingress work.

Reply:
Hi thanks for looking at the PR :)

I figured there was some reasoning behind the missing content in the `helm package` output. This solves it, but it sounds like you have more experience than I do with it. What are the downsides of including this?

Reply:
If it's not needed for ingress, I think it should not be included in the context of this PR regardless of what this does.

Reply:
@briantopping could we have this removed please? Merging this would be nice :)

Reply:
Removed, apologies for delay.